src/benchmark/tpch_data_micro_benchmark.cpp

```diff
@@ -29,7 +29,7 @@ class TableWrapper;
 // Defining the base fixture class
 class TPCHDataMicroBenchmarkFixture : public MicroBenchmarkBasicFixture {
  public:
-  void SetUp(::benchmark::State& state) {
+  void SetUp(::benchmark::State& state) override {
```

Comment:
clang-11 finally complains about virtual overrides not marked as either override or final :)